doc/ref_arch/kubernetes/chapters/appendix-a.md

```diff
@@ -1,18 +1,26 @@
 [<< Back](../../kubernetes)
-# Appendix A - Guidance for VNF-to-CNF Transition
+# Appendix A - Guidelines For VNF Vendors
 <p align="right"><img src="../figures/bogo_ifo.png" alt="scope" title="Scope" width="35%"/></p>
 
 ## Table of Contents
-* [A.1 TBC](#A.1)
-* [A.2 TBC](#A.2)
+
+* [A.1 Exception List](#A.1)
+* [A.2 Guidance for VNF-to-CNF Transition](#A.2)
 * [A.3 TBC](#A.3)
 * [A.4 TBC](#A.3)
 
 <a name="A.1"></a>
-## A.1 TBC
+## A.1 Exception List
+
+| Ref         | Type       | Name               | RM Exception | Details |
+|-------------|------------|--------------------|--------------|---------|
+| ra2.exc.001 | Technology | SR-IOV             | rm.exc.001   | This exception allows workloads (VNF/CNF) to use SR-IOV Pass-Through. RA2 architectures allows plugins such as SR-IOV CNI to be used. |
```

Comment:
The exception in RM Appendix A is:
>This exception allows workloads (VNF/CNF) to rely on SR-IOV technology.

It doesn't include the Pass-Through part. My understanding was that the PCI-Passthrough part was the issue with SR-IOV as it ties a workload to a particular VF on a particular PCIe card. Therefore, should we remove that part from here?

Reply:
Humm, interesting. I believe I've copied the text from RM Appendix A. 
In case of containers SR-IOV means PCI-Passthrough as the VF-s are mounted to the containers as PCI devices by the device plugin. I can remove the PCI-Passthrough part and add, refer to the usage of sriov-network-device-plugin. Is that OK?

Reply:
Yes I would prefer that.  I agree that, for containers, the tying of a particular workload (container) to a particular PCIe card (hardware) isn't necessarily a problem as we don't move containers in the same way as we do VMs - they are re-scheduled and so the Kubernetes device plugin is the correct method.

Reply:
Okey, done in 36a6b7f.